packages/web-runtime/src/store/config.js

```diff
@@ -109,7 +118,7 @@ const getters = {
     return parsed
   },
   theme: (state) => {
```

Comment:
Is the theme getter used at all? You were accessing the theme with `configuration.currentTheme` everywhere. If that's intended you might want to drop this getter entirely. Or map & use it instead of `configuration`.